internal/store/leaf/recipients.go

```diff
@@ -270,7 +270,8 @@ func (s *Store) ensureOurKeyID(ctx context.Context, rs []string) []string {
 // OurKeyID returns the key fingprint this user can use to access the store
 // (if any).
 func (s *Store) OurKeyID(ctx context.Context) string {
-	for _, r := range s.Recipients(ctx) {
+	recp := s.Recipients(ctx)
```

Comment:
Can you add a check in the GPG Recipients() to specifically call out "make sure your key is trusted by gpg". This will save people from a rabbit hole of figuring out why this error occurs.